pyproject.toml

```diff
@@ -59,6 +59,7 @@ all = [
     "lancedb>=0.2",
     "pyarrow",
     "pymupdf>=1.23.6",
+    "python-docx>=1.1.0",
```

Comment:
Any reason for this pin? If not, I would prefer to leave it unpinned to give the solver more flexibility.